Readability.js

```diff
@@ -1316,6 +1316,61 @@ Readability.prototype = {
     return metadata;
   },
 
+  /**
+   * Find all <noscript> that located after <img> node, and contains exactly
+   * single <img> element. Once it found, this method will replace the previous <img>
+   * with <img> inside <noscript>, then finally remove the <noscript> tag. This is
+   * done because in some website (e.g. Medium), they use lazy load method like this.
+   *
```

Comment:
Some English nits here. Can you use:

```
Find all <noscript> that are located after <img> nodes, and which contain only one <img> element.
Replace the first image with the image from inside the <noscript> tag, and remove the <noscript> tag.
This improves the quality of the images we use on some sites (e.g. Medium).
```

Reply:
Fixed in [dc33dd6](https://github.com/mozilla/readability/pull/588/commits/dc33dd69793dc0af6e80b02e68af84a1609dd31b)